iceberg/src/test/scala/org/apache/spark/sql/delta/ConvertToIcebergSuite.scala

```diff
@@ -123,21 +128,160 @@ class ConvertToIcebergSuite extends QueryTest with Eventually {
         withDefaultTablePropsInSQLConf {
           deltaSpark.range(10).write.format("delta")
             .option("path", testTablePath)
+            .option("delta.enableIcebergCompatV2", "true")
             .saveAsTable(testTableName)
         }
       }
       withDeltaSparkSession { deltaSpark =>
         deltaSpark.range(10, 20, 1)
           .write.format("delta").mode("append")
           .option("path", testTablePath)
+          .option("delta.enableIcebergCompatV2", "true")
           .saveAsTable(testTableName)
       }
       verifyReadWithIceberg(testTableName, 0 to 19 map (Row(_)))
     }
   }
 
-  def runDeltaSql(sqlStr: String): Unit = {
+  test("Expire Snapshots") {
```

Comment:
put this case in spark/src/test/scala/org/apache/spark/sql/delta/UniversalFormatSuiteBase.scala